rdrf/report/TrrfGraphQLView.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from graphene_django.views import GraphQLView
+from graphql import GraphQLError
+from django.utils.translation import ugettext_lazy as _
+
+logger = logging.getLogger(__name__)
+
+
+class PublicGraphQLError(GraphQLError):
```

Comment:
Happy to take suggestions for naming here.

Reply:
Fine with me :+1: